deploy/src/main/sandbox/hono-values.yml

```diff
@@ -224,39 +225,31 @@ kafka:
   ## @param logSegmentBytes The maximum size of a log segment file. When this size is reached a new log segment will be created
   ## 10MB
   logSegmentBytes: _10485760
```

Comment:
The `kafka.log[*]` properties here are not supported anymore since bitnami/kafka chart version 24.0.
See the [To 24.0.0](https://github.com/bitnami/charts/tree/main/bitnami/kafka#to-2400) section in the chart README:
> Due to the implementation of readOnlyRootFs support, the following settings have been removed and will now rely on Kafka defaults. To override them, please use `extraConfig` to extend your Kafka configuration instead.
> [...]
>    logFlushIntervalMessages
>    logFlushIntervalMs
>    logRetentionBytes
>    logRetentionCheckIntervalMs
>    logRetentionHours

Reply:
The configuration would look like this:
```yaml
kafka:
  broker:
    extraConfig: |
      log.flush.interval.messages=1000
      log.flush.interval.ms=1000
      log.retention.bytes=10485760
      log.retention.check.interval.ms=300000
      log.retention.hours=1
      log.segment.bytes=10485760
```

Reply:
I did a last test and saw that these properties actually didn't show up in the kafka pod.
It turns out the `extraConfig` below `controller` must be set here, not below `broker`.
Together with the other `extraConfig` content already defined in the Hono chart, this results in:
```yaml
kafka:
  controller:
    extraConfig: |
      group.initial.rebalance.delay.ms=0
      offsets.topic.replication.factor=1
      transaction.state.log.min.isr=1
      transaction.state.log.replication.factor=1
      log.flush.interval.messages=1000
      log.flush.interval.ms=1000
      log.retention.bytes=10485760
      log.retention.check.interval.ms=300000
      log.retention.hours=1
      log.segment.bytes=10485760
```

Reply:
Yeah, that's what I reckoned as well while going through the docs. I will update accordingly ...